pkg/api/v1/persistentvolume/util.go

```diff
@@ -129,6 +131,14 @@ func VisitPVSecretNames(pv *corev1.PersistentVolume, visitor Visitor) bool {
 				return false
 			}
 		}
+		if utilfeature.DefaultFeatureGate.Enabled(features.ExpandCSIVolumes) {
+			if source.CSI.ControllerExpandSecretRef != nil {
```

Comment:
nit: move this above the ControllerPublishSecretRef block so it's alphabetical and the controller secrets are together